app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class RecipesController < ApplicationController
+  load_and_authorize_resource
+
+  def index
+    @recipes = current_user.recipes.all
+  end
+
+  def show
```

Comment:
- [ ] Kindly make sure that the recipe details have a toggle button to make the recipe public or private as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=Should%20display%20a%20toggle,as%20in%20the%20wireframe.) now when I try to create a private recipe I get this error.
![image](https://github.com/lorenzoworx/chef-chapters/assets/24703171/b3f95d7d-0444-4993-9517-03a8a4041cd9)

Reply:
- [ ] well-done adding the toggle as requested by the previous reviewer, but currently the toggle does not work as expected, when I toggle a recipe to public and go back to the recipe list, clicking on the recipe again, I discover that what I toggled did not change, kindly fix that

Reply:
done